packages/terra-table/src/SingleSelectableRows.jsx

```diff
@@ -60,10 +65,26 @@ class SingleSelectableRows extends React.Component {
     };
   }
 
-  newPropsForRow(row, index, onClick) {
+  wrappedOnKeyDownForRow(row, index) {
+    const initialOnKeyDown = this.props.onKeyDown;
+
+    return (event) => {
+      if (event.nativeEvent.keyCode === Number(13)) {
+        if (row.props.isSelectable !== false && this.shouldHandleSelection(index)) {
+          this.handleSelection(event, index);
+        }
+
+        if (initialOnKeyDown) {
+          initialOnKeyDown(event);
+        }
```

Comment:
react-portal has the same sort of logic here: https://github.com/tajo/react-portal/blob/master/lib/portal.js#L116-L120

It might be a good example to reference.